src/main/kotlin/no/nav/sifinnsynapi/konsument/pleiepenger/syktbarn/PleiepengersøknadUtils.kt

```diff
@@ -23,22 +25,29 @@ internal object PleiepengersøknadKeysV1 {
 }
 
 internal object PleiepengerJSONObjectUtils {
+
     fun JSONObject.finnOrganisasjon(søknad: SøknadDAO, organisasjonsnummer: String): JSONObject {
-        val organisasjoner = getJSONObject(PleiepengersøknadKeysV1.ARBEIDSGIVERE).getJSONArray(PleiepengersøknadKeysV1.ORGANISASJONER)
-        var funnetOrg: JSONObject? = null
+        val organisasjonerObjekt = get(ARBEIDSGIVERE)
+        val organisasjoner = when (organisasjonerObjekt) {
+                is JSONObject -> organisasjonerObjekt.getJSONArray(PleiepengersøknadKeysV1.ORGANISASJONER)
+                is JSONArray -> organisasjonerObjekt
+                else -> throw Error("Feil JSONObjekt for feltet $ARBEIDSGIVERE") // TODO: 01/10/2021 Lage egen exception?
```

Comment:
Er ikke nødvendig med egen exception, men kan forbedre selve feilmeldingen litt.
_"Ugyldig type for feltet $ARBEIDSGIVERE. Forventet enten JSONObject eller JSONArray, men fikk ${organisasjonerObjekt.javaClass}"_

Reply:
Nice, done